package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "justice-ui-library",
-  "version": "0.1.37",
+  "version": "0.1.39",
```

Comment:
i think for the similar case in the future, we can remove the tag from last PR (0.1.38), then change this to 0.1.38 so that we only bump the tag once